tetris.py

```diff
@@ -22,6 +23,8 @@
 
 block_letters = ["I", "L", "L_rev", "O", "E", "Z", "Z_rev"]
 
+ENUM_game_status = Enum('game_status', 'in_progress, game_lost, game_won')
```

Comment:
just follow the fucking naming convention

```suggestion
GameStatus = Enum('GameStatus', 'in_progress, game_lost, game_won')
```